### DISCUSSION TITLE
Jamulus News on Homepage

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#2426] [Ideas]
Missing a "News" area on 

https://jamulus.io

* Latest Version 
* ...

Target Group: No-IT Musicians

**Comment by gilgongo** [comment]
Hi @reinhardwh - thanks for suggesting this. What sort of stuff would you like to see in such a "News" area?

**Reply by reinhardwh** [reply]
Hello Gilgongo

The Audacity page https://www.audacityteam.org/ page could be an inspiration:

![image](https://user-images.githubusercontent.com/77500932/155354395-2008ade4-ca3d-4803-8626-cddece05ed37.png)

![image](https://user-images.githubusercontent.com/77500932/155354424-ac36306a-9830-45cb-8d8e-8150feb040d5.png)

**Reply by ann0see** [reply]
In theory we could use the blog feature of jekyll (like we use for the community KB). Or we could of course rename the KB to something blog-ish?

**Reply by gilgongo** [reply]
Could re-cast the KB as the blog I guess (assuming we have the ability to write content for it regularly, which isn't certain). But a lot of this is more about navigation and IA, and feels like a bigger conversation about the website. To be fair, we have talked about this in the past (we need download pages and stuff). But lack the resources and skills really...
